composer.json

```diff
@@ -20,10 +20,7 @@
         "jakub-onderka/php-console-highlighter": "0.3.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7|~4.0",
-        "symfony/finder": "~2.1",
-        "squizlabs/php_codesniffer": "~2.0",
-        "fabpot/php-cs-fixer": "~1.3"
```

Comment:
Me too — save for when new rules are introduced between builds, see FriendsOfPHP/PHP-CS-Fixer#1038 — but most people don't.

It feels like that's moving in the wrong direction. It's fine for PHPUnit, because it's old enough to be almost completely stable and backwards compatible. php-cs-fixer is such a moving target that just using whatever version someone happens to have globally installed is _almost guaranteed_ to be wrong.